packages/@sanity/types/src/schema/definition/schemaDefinition.ts

```diff
@@ -0,0 +1,131 @@
+import {PreviewConfig} from '../preview'
```

Comment:
Hm, actually – this file is already in a directory called `definition`. Should this file be renamed to `types.ts` then?

Reply:
I have no strong opinion on it.